test/nullablevector.jl

```diff
@@ -152,9 +155,21 @@ module TestNullableVector
                   NullableArray([2:12...]))
     @test isequal(append!(X, [Nullable(13), Nullable(14)]),
                   NullableArray([2:14...]))
-    @test isequal(append!(X, [Nullable(15), Nullable()]),
+    @test isequal(append!(X, [Nullable(15), Nullable{Int64}()]),
                   NullableArray([2:16...], vcat(fill(false, 14), true)))
 
+    #--- test Base.prepend!
+
+    # Base.prepend!(X::NullableVector, items::AbstractVector)
+
+    XX = NullableArray([3:12...])
+    @test isequal(prepend!(XX, [1, 2]),
+                  NullableArray([1:12...]))
+    @test isequal(prepend!(XX, [Nullable(-1), Nullable(0)]),
+                  NullableArray([-1:12...]))
+    @test isequal(prepend!(XX, [Nullable{Int64}(), Nullable(-2)]),
+                  NullableArray([-3:12...], vcat(true, fill(false, 15))))
```

Comment:
Even on 0.4, you can write this as `NullableArray(Nullable{Int}[Nullable(), -2:12...])`, which is more readable.